scripts/js/lib/api/processHtml.ts

```diff
@@ -94,8 +95,14 @@ export function loadImages(
       const $img = $(img);
 
       const fileName = $img.attr("src")!.split("/").pop()!;
+      const fileExtension = path.extname(fileName);
+
+      // Remove `false &&` when ready to switch to AVIF
+      let dest =
+        false && [".png", ".jpg", ".jpeg"].includes(fileExtension)
+          ? `${imageDestination}/${path.basename(fileName, fileExtension)}.avif`
+          : `${imageDestination}/${fileName}`;
```

Comment:
Feel free to ignore. Motivations for this suggestion:
- make the feature gate more explicit
- add a comment about why we convert to AVIF. It'd be good to have this comment even after removing the feature gate

```suggestion
      // Remove `false &&` when ready to switch to AVIF
      const avif_feature_gate_enabled = false
      let dest;
      if (avif_feature_gate_enabled) {
         // We convert PNG and JPG to AVIF for reduced file size.
        dest = [".png", ".jpg", ".jpeg"].includes(fileExtension)
          ? `${imageDestination}/${path.basename(fileName, fileExtension)}.avif`
          : `${imageDestination}/${fileName}`;
      } else {
       dest = `${imageDestination}/${fileName}`;
      }
```

The formatting is probably wrong

Reply:
I would usually agree with the named bools, but I think since the feature gate code will only exist for a few days it's better to avoid the extra nesting. Comment is a good idea though.